CDP4Addin/Addin.cs

```diff
@@ -109,17 +111,46 @@ public Addin()
         {
             logger.Debug("starting CDP4-CE addin");
 
+            var appSettings = new AppSettingsService();
+            this.ReadAppSettings(appSettings);
+
             this.PreloadAssemblies();
             this.RedirectAssemblies();
             this.SetupIdtExtensibility2Events();
             this.SetupEventListeners();
 
             // Set the Theme of the application
-            DevExpress.Xpf.Core.ThemeManager.ApplicationThemeName = DevExpress.Xpf.Core.Theme.SevenName;
+            ThemeManager.ApplicationThemeName = Theme.SevenName;
 
             logger.Debug("CDP4-CE addin started");
         }
-        
+
+        /// <summary>
+        /// Reads the app settings from disk
+        /// </summary>
+        internal AddinSettings ReadAppSettings(IAppSettingsService appSettingService)
```

Comment:
in the constructor you are reading the appsettings (and it needs to go to Mef initialization), but you are not doing anything with the returned settings? What is the intent, not clear to me.

Reply:
Right now this is just the hookup. The actual use of the settings will be in the next issue.